src/components/swap/styleds.tsx

```diff
@@ -23,7 +24,9 @@ export const PageWrapper = styled.div`
 `
 
 // Mostly copied from `AppBody` but it was getting too hard to maintain backwards compatibility.
-export const SwapWrapper = styled.main<{ margin?: string; maxWidth?: string }>`
+export const SwapWrapper = styled.main`
+  ${glowEffect}
```

Comment:
why do we have glowEffect and also a component GlowEffect?

Reply:
So, if you already have a Styled component or don't want to wrap your components in yet another `div`, `glowEffect` (plain style) comes into play. This makes it much easier to add a glow effect and avoid introducing any unwanted side effects as a result of wrapping a layout in a `div`. You can't also do `styled(GlowEffect)` in this particular scenario, because we're using `main` and not a `div`. 

I left GlowEffect component as an alternative for components that are either:
- not using Styled Components (see Widget), and we would have to create a component for them anyway
- it is not possible to apply a style to a component because it already utilizes `box-shadow` or some other properties (e.g. many components such as Table use a `1px` box-shadow to show a border)

Reply:
Removed component